packages/internals/src/__tests__/engine-commands/formatSchema.test.ts

```diff
@@ -12,7 +12,7 @@ if (process.env.CI) {
 
 const ctx = jestContext.new().add(jestConsoleContext()).assemble()
 
-describe('format wasm', () => {
+describe('schema wasm', () => {
```

Comment:
doesn't this refer to the action of formatting?

Reply:
Yes, here actually a better name would be "format" but actually these tests are now duplicates of the ones below, and we could clean up there, though not urgent at all.